sys/kern/uipc_mbuf.c

```diff
@@ -171,6 +173,10 @@ CTASSERT(offsetof(struct mbuf, m_pktdat) % 8 == 0);
 CTASSERT(offsetof(struct mbuf, m_dat) == 32);
 CTASSERT(sizeof(struct pkthdr) == 56);
 CTASSERT(sizeof(struct m_ext) == 160);
+#elif defined(__CHERI_PURE_CAPABILITY__)
```

Comment:
We've had hacks to define LP64 things in the past; probably best to check __CHERI_PURE_CAPABILITY__ first.